deploy/sample/ml-config.sample.xml

```diff
@@ -458,6 +458,12 @@
       </collections>
       <privileges>
         <privilege>
+          <privilege-name>any-uri</privilege-name>
```

Comment:
Ack. Yes, it does. I'll move that and run another comparison between the ml-config.xml I tested with and this file.